### PR TITLE
Ss gatk base recalibrator 4.1.2.0

### PR DESCRIPTION
**Pull request by shalabhsuman**
#30

### CODE REVIEW

File: gatk_BaseRecalibrator_4.1.2.0/gatk_baserecalibrator.cwl [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+      One or more databases of known polymorphic sites used to exclude regions
+      around known polymorphisms from analysis. This second one is optional.
+    secondaryFiles:
+      - .fai
+      - ^.dict
+outputs:
+  - id: output
+    type: File
+    outputBinding:
+      glob: '$(inputs.input.basename.replace(''.bam'', '''')).recal.table'
```
Comment (rhshah) [inline gatk_BaseRecalibrator_4.1.2.0/gatk_baserecalibrator.cwl]: - [ ] please change the file name to be `_bqsr.table`

File: gatk_BaseRecalibrator_4.1.2.0/gatk_baserecalibrator.cwl [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+arguments:
+  - position: 0
+    prefix: ''
+    valueFrom: BaseRecalibrator
+  - position: 0
+    prefix: '--tmp-dir'
+    valueFrom: .
+  - position: 0
+    prefix: '--output'
+    valueFrom: '$(inputs.input.basename.replace(''.bam'', '''')).recal.table'
```
Comment (rhshah) [inline gatk_BaseRecalibrator_4.1.2.0/gatk_baserecalibrator.cwl]: - [ ] please change the file name to be `_bqsr.table`

File: gatk_BaseRecalibrator_4.1.2.0/gatk_baserecalibrator.cwl [hunk 3]
```diff
@@ -0,0 +1,284 @@
+class: CommandLineTool
+cwlVersion: v1.0
+id: gatk_base_recalibrator_4_1_2_0
+baseCommand:
```
Comment (rhshah) [inline gatk_BaseRecalibrator_4.1.2.0/gatk_baserecalibrator.cwl]: Can you also add the additional namespaces and the contributor tags

**Comment by shalabhsuman** [comment]
I looked at that and making it part of `Inputs` was giving issues with the Javascript and CWLTool. So I am going to leave it as part of `arguments` for now.

**Comment by rhshah** [comment]
@shalabhsuman  Can you please elaborate on what issues?

**Pull request merged**